cmd/kube-apiserver/app/options/globalflags_legacyproviders.go

```diff
@@ -0,0 +1,29 @@
+// +build !nolegacyproviders
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+
+	"k8s.io/component-base/cli/globalflag"
+)
+
+func registerLegacyGlobalFlags(fs *pflag.FlagSet) {
+	globalflag.Register(fs, "cloud-provider-gce-lb-src-cidrs")
```

Comment:
this should also probably be marked deprecated like the matching flag in kube-controller-manager, but that seemed somewhat orthogonal to this PR

Reply:
See https://github.com/kubernetes/kubernetes/pull/73509#discussion_r305561794